### PR TITLE
[Fix] caching walletMode call

### PR DESCRIPTION
**Pull request by makylfang**
# Pull Request Template

## Changes

- Caching wallet mode call

## Checklist

- [ ] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: src/pages/loggedInView.vue [hunk 1]
```diff
@@ -120,9 +120,11 @@ onMounted(async () => {
       userStore.privateKey = await getPrivateKey(userStore.privateKey)
     }
     devLogger.log('[loggedInView] after keygen', userStore.privateKey)
-    await setRpcConfigs()
-    await getRpcConfig()
-    await getKeySpaceType()
+
+    await Promise.all([
+      setRpcConfigs().then(async () => await getRpcConfig()),
```
Comment (mmjee) [inline src/pages/loggedInView.vue:125]: `() => getRpcConfig()` would do the same and be less expensive than instantiating an async function and would save a few bytes

**Review by mmjee — APPROVED** [review]

File: src/pages/loggedInView.vue (hunk 1 above)
Comment (makylfang) [reply src/pages/loggedInView.vue]: @mmjee fixed.

File: src/pages/loggedInView.vue [hunk 2]
```diff
@@ -137,18 +139,19 @@ onMounted(async () => {
...
-      await requestHandler.setRpcConfig({
-        chainId: selectedChainId,
-        ...rpcConfig,
-      })
+      requestHandler
+        .setRpcConfig({
+          chainId: selectedChainId,
+          ...rpcConfig,
+        })
+        .then(async () => await requestHandler.sendConnect())
```
Comment (mmjee) [inline src/pages/loggedInView.vue:147]:
```suggestion
        .then(() => requestHandler.sendConnect())
```

Comment (makylfang) [reply src/pages/loggedInView.vue]: @mmjee done

**Review by shrinathprabhu — APPROVED** [review]

**Pull request merged**